.github/workflows/pr-test.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests
+
+name: PR Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
please update to use pinned hashes `@v3` should be `@962678f22cdec338fba2a277a33cf81d5e28f7b9` i believe. but double check the upsteam